deployments/local-setup-instance/provisioning/run-clients.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+echo "Running Core and ECDSA clients..."
+
+cd local-setup
+
+WORKDIR=$(pwd)
+
+sudo systemd-run --unit=keep-core-1 -p WorkingDirectory=$WORKDIR ./run-core-1.sh
```

Comment:
How much does it hurt to run two core clients? Not something abtolutely necessary, just asking.

Reply:
This is rather simple regarding changes in the code. Some pain can emerge in terms of resource usage. Didn't checked this in detail but the whole `local-setup` instance is already huge so I think we should have good reasons to start more `keep-core` clients.